haystack/environment.py

```diff
@@ -18,6 +18,12 @@
 env_meta_data: Dict[str, Any] = {}
 
 
+def set_pytorch_secure_model_loading(flag_val="1"):
+    # To load secure only model pytorch requires value of
+    # TORCH_FORCE_WEIGHTS_ONLY_LOAD to be ["1", "y", "yes", "true"]
+    os.environ["TORCH_FORCE_WEIGHTS_ONLY_LOAD"] = flag_val
```

Comment:
should we check if by any chance the user has already set the env var, and do nothing in case? Something like (pseudo-code):
```python
if os.getenv("TORCH_FORCE_WEIGHTS_ONLY_LOAD") is None:
  # do it
else:
  # log that TORCH_FORCE_WEIGHTS_ONLY_LOAD is already set and Haystack will use its value as it is
```

Reply:
Added appropriate log, please review if it makes sense.